ck.sh

```diff
@@ -228,10 +233,11 @@ case "$COMMAND" in
 				echo
 				echo "Here are the steps to follow:"
 				echo
-				echo "1. Navigate to the ckeditor-dev submodule directory ('cd ckeditor-dev')"
+				echo "1. Navigate to the ckeditor-dev submodule directory (\`cd ckeditor-dev\`)"
 				echo "2. Work on your changes"
 				echo "3. Commit your changes"
-				echo "4. Run 'sh ck.sh patch' to generate the patches"
+				echo "4. Return to the repository root (\`cd ..\`)"
+				echo "5. Run 'sh ck.sh patch' to generate updated patches"
```

Comment:
Much clearer yes, now the only thing we need to be "cool" is emoji support (just kidding)

Reply:
![deadly-serious](https://user-images.githubusercontent.com/7074/79466065-5e5f2000-7ffc-11ea-8c41-6d0ef65e78d0.gif)